Packages/Tracking/Core/Runtime/Scripts/LeapProvider.cs

```diff
@@ -80,6 +82,14 @@ protected void DispatchFixedFrameEvent(Frame frame)
 
     }
 
+    public enum TrackingSource
```

Comment:
Would benefit from some comments around each value, even though the name are pretty good.